onadata/libs/utils/export_builder.py

```diff
@@ -1207,7 +1207,7 @@ def _is_numeric(xpath, element_type, data_dictionary):
                 # check if it is a choice part of multiple choice
                 # type is likely empty string, split multi select is binary
                 element = data_dictionary.get_element(xpath)
-                if element.type == SELECT_ONE:
+                if element and element.type == SELECT_ONE:
```

Comment:
Should we add a test to ensure the code `is_all_numeric` is not called if element is none?

Reply:
It should be called; the element could be null when the parent is a select multiple

Reply:
The function checks the parent element in the next few lines

Reply:
What scenario would cause it to be called ? I don't think this scenario in particular requires a test...

Reply:
Much like line `1218`; Element are returned as `None` even when in sections in very rare cases... _When the form has a very large choice list e.t.c_